airflow/contrib/operators/kubernetes_pod_operator.py

```diff
@@ -71,7 +71,8 @@ class KubernetesPodOperator(BaseOperator):
     :type affinity: dict
     :param node_selectors: A dict containing a group of scheduling rules
     :type node_selectors: dict
-    :param config_file: The path to the Kubernetes config file
+    :param config_file: The path to the Kubernetes config file.
+        If not specified, defaults to ``~/.kube/config``
```

Comment:
```suggestion
        If not specified, using defaults value ``~/.kube/config``
```

Reply:
I adjusted to a modified version of your suggestion that has more correct grammar - PTAL once travis runs :)

Reply:
Looking great.

Reply:
Great. Let me know if anything else needs to be done to make it merge-able! :)

Reply:
You should remind committer to review your change.

Reply:
@XD-DENG @potiuk PTAL if you have space time. thx